lib/bootstrap.js

```diff
@@ -280,6 +280,34 @@ function bootstrapError(component, e) {
   return bluebird.reject();
 }
 
+/**
+ * Bootstraps a site, and if it fails, falls back to the parent sites' bootstrap (for subsites)
+ *
+ * @param {Object} site
+ * @return {Promise}
+ */
+function bootstrapSite(site) {
+  // fetches itself if it is the main site
+  const parent = siteService.getParentSite(site.slug);
+
+  // using module.exports for test
+  return module.exports.bootstrapPath(parent.dir, site)
+    .catch(function () {
```

Comment:
Let's replace this with arrow function
```suggestion
    .catch(() => {
```